src/environments.rs

```diff
@@ -76,14 +84,27 @@ impl Environments {
         }
     }
 
-    pub fn get_environment_names(&self, org_id: Option<&str>) -> GraphQLResult<Vec<String>> {
+    pub fn get_environment_details(
+        &self,
+        org_id: Option<&str>,
+    ) -> GraphQLResult<Vec<EnvironmentDetails>> {
         let environments = self.get_environments_full(org_id)?;
-        let mut list = environments
-            .into_iter()
-            .map(|n| n.name)
-            .collect::<Vec<String>>();
-        list.sort();
-
-        Ok(list)
+        let mut env_info: Vec<EnvironmentDetails> = Vec::new();
```

Comment:
Just in case you're unaware, there's a [vec!](https://doc.rust-lang.org/std/macro.vec.html) macro you can use as well.

Reply:
Thanks for the tip... I will explore that in the future.